store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -220,8 +220,6 @@ public String getGroupFromColumnFamily(final byte[] columnFamily) {
                     } catch (final SerialisationException e) {
                         throw new AccumuloElementConversionException(e.getMessage(), e);
                     }
-                } else {
-                    rtn = serialiser.serialiseNull();
```

Comment:
So if there is no property, it uses empty byte ?

Reply:
Yes, this is a corner case where if visibility is missing (null) or empty string, it will still be deserialised as empty string.